src/main/java/hudson/plugins/ansicolor/AnsiOutputStream.java

```diff
@@ -42,7 +42,7 @@
  */
 public class AnsiOutputStream extends FilterOutputStream {
 
-    public static final byte[] REST_CODE = resetCode();
+    private static final byte[] REST_CODE = resetCode();
```

Comment:
Just a hint: Please note, that `AnsiOutputStream.java` is copied from [jansi](https://github.com/fusesource/jansi/blob/master/jansi/src/main/java/org/fusesource/jansi/AnsiOutputStream.java) because of version conflict (see also #95 and #96).
Doing thus changes will make it harder to compare against the original in future.